packages/manager/modules/pci/src/components/project/storages/databases/backup.class.js

```diff
@@ -0,0 +1,42 @@
+import includes from 'lodash/includes';
+import 'moment';
+
+export default class Backup {
+  constructor({ createdAt, id, status, description, size }) {
+    Object.assign(this, {
+      createdAt,
+      id,
+      status,
+      description,
+      size,
+    });
+  }
+
+  get expirationDate() {
+    return moment(this.createdAt)
+      .add(90, 'days')
+      .format();
+  }
+
+  get statusGroup() {
+    if (
+      includes(['PENDING', 'CREATING', 'UPDATING', 'DELETING'], this.status)
+    ) {
+      return 'PENDING';
+    }
+
+    if (includes(['ERROR', 'ERROR_INCONSISTENT_SPEC'], this.status)) {
+      return 'ERROR';
+    }
+
+    if (includes(['READY'], this.status)) {
+      return 'ACTIVE';
+    }
+
```

Comment:
This could be done by a `switch` case, which could also avoid use of `includes`. (nitpick also on constants for status, which can be set directly in this class file if used only here)